lib/node_modules/@stdlib/assert/deep-has-own-property/docs/repl.txt

```diff
@@ -19,21 +19,21 @@
     Returns
     -------
     bool: boolean
-        Boolean indicating if an obj has a specified path.
+        Boolean indicating if an object has a specified path.
 
     Examples
     --------
     > var obj = { 'a': { 'b': { 'c': 'd' } } };
-    > var bool = {{alias}}( obj, 'a.b.c' )
+    > var bool = {{alias}}( obj, 'a.b.c' );
```

Comment:
You were right to leave off the semicolon. :) In the REPL, semicolons have meaning. Similar to MATLAB and Julia, if a semicolon is present on a line, it means to suppress printed output. If a semicolon is absent, we print output. Here, as we are printing output, we leave off the semicolon.

Reply:
At first I was thinking like "the semicolon should not be present here".
But then again, I thought maybe you missed it :).

I'm gonna take the appropriate changes you've outlined and run the lint/tests hooks again, then pass it to you to review.